### PR TITLE
Export API with Json Output and Add missing Api /apis/operations/policies

### PR DESCRIPTION
**Pull request by solankisamir**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by solankisamir** [comment]
@jianghaolu can you take a look?

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json [hunk 1]
```diff
@@ -1056,7 +1104,96 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}?export=true": {
```
Comment (jianghaolu) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: If your request/response differs if an optional query parameter is provided, please use extension `x-ms-path`: "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json (hunk 1 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: @jianghaolu, I am already using the correct extension [x-ms-paths](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-paths). There is no extension `x-ms-path`

Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: @olydis, can you take a look at the PR? This is the second iteration of https://github.com/Azure/azure-rest-api-specs/pull/1554

Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: @olydis, can you take a look at the PR? This is the second iteration of https://github.com/Azure/azure-rest-api-specs/pull/1554

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json [hunk 2]
```diff
@@ -1056,7 +1104,96 @@
...
+        }
+      }
+    }
+  },
   "definitions": {
+    "ApiExportResult": {
+      "properties": {
+        "link": {
+          "type": "string",
+          "description": "Link to the Storage Blob containing the result of the export operation. The Blob Uri is only valid for 5 minutes."
```
Comment (solankisamir) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: @ravbhatnagar  we implemented the change we discussed in https://github.com/Azure/azure-rest-api-specs/pull/1554#issuecomment-324672141

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json (hunk 1 above)
Comment (jianghaolu) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: You're right. My bad.

**Review by jianghaolu — APPROVED** [review]

**Comment by solankisamir** [comment]
#sign-off

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json [hunk 3]
```diff
@@ -1056,7 +1104,96 @@
...
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "Accept",
```
Comment (jianghaolu) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: Please use camel case for parameter name.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json (hunk 3 above)
Comment (solankisamir) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json]: That is a standard header as defined by the [RFC](https://tools.ietf.org/html/rfc7231#section-5.3.2)

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**